db/app.js

```diff
@@ -0,0 +1,13 @@
+const exp = require("express")
+const app = exp()
+const {getApi} = require("./controller")
+module.exports = app
+
+app.get("/api", getApi)
+
+app.use((err, req, res, next) => {
+
+  const status = err.status || 500;
```

Comment:
Consider in this error handler if either one of status or code is undefined we'd send something like e.g. 404 - internal server error which will confuse the user! Refer back to the error handling notes to set up this middleware more efficiently and follow good separation of concerns: https://l2c.northcoders.com/courses/sd-notes/back-end#sectionId=error-handling,step=intro